lib/t/accounts.ex

```diff
@@ -114,6 +114,17 @@ defmodule T.Accounts do
     |> Repo.update_all(set: [last_active: time])
   end
 
+  def update_location(user_id, location) do
+    primary_rpc(__MODULE__, :local_update_location, [user_id, location])
```

Comment:
For later: `primary_rpc` only if `new_location != old_location (from read replica)`.